docs/aggregation.ipynb

```diff
@@ -1,260 +1,294 @@
-     }
-    ],
-    "source": [
-     "import numpy as np\n",
-     "import xarray_beam as xbeam\n",
-     "import xarray\n",
-     "\n",
-     "datasets = [\n",
-     "    xarray.Dataset({'foo': ('x', np.random.randn(3))})\n",
-     "    for _ in range(100)\n",
-     "]\n",
-     "datasets | xbeam.Mean.Globally()"
-    ]
-   },
-   {
-    "cell_type": "markdown",
-    "id": "5670130b",
-    "metadata": {},
-    "source": [
-     "Notice how existing dimensions on each datasets are unchanged by the transformation. If you want to average over existing dimension, you would need to do that aggregation yourself, e.g., by averaging inside each chunk before combining the data."
-    ]
-   },
-   {
-    "cell_type": "markdown",
-    "id": "6ab152b6",
-    "metadata": {},
-    "source": [
-     "Similarly, the keys fed into `xbeam.Mean.PerKey` can be any hashables, including but not limited to `xbeam.Key`:"
-    ]
-   },
-   {
-    "cell_type": "code",
-    "execution_count": 2,
-    "id": "b2399483",
-    "metadata": {},
-    "outputs": [
-     {
-      "name": "stderr",
-      "output_type": "stream",
-      "text": [
-       "WARNING:apache_beam.options.pipeline_options:Discarding unparseable args: ['/Users/shoyer/miniconda3/envs/xarray-beam/lib/python3.9/site-packages/ipykernel_launcher.py', '-f', '/Users/shoyer/Library/Jupyter/runtime/kernel-e19c251a-dc50-467c-aaed-1252152cd403.json']\n"
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "aaf2f2cb",
+   "metadata": {},
+   "source": [
+    "# Aggregation\n",
+    "\n",
+    "Xarray-Beam can perform efficient distributed data aggregation in the \"map-reduce\" model. \n",
+    "\n",
+    "This currently only includes `Mean`, but we would welcome contributions of other aggregation functions such as `Sum`, `Std`, `Var`, `Min`, `Max`, etc."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b77b953",
+   "metadata": {},
+   "source": [
+    "## High-level API\n",
+    "\n",
+    "The `Mean` transformation comes in three forms:  {py:class}`Mean <xarray_beam.Mean>`, {py:class}`Mean.Globally <xarray_beam.Mean.Globally>`, and {py:class}`Mean.PerKey <xarray_beam.Mean.PerKey>`. The implementation is highly scalable, based on a Beam's [`CombineFn`](https://beam.apache.org/documentation/transforms/python/aggregation/combineglobally/#example-4-combining-with-a-combinefn)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39719ac3",
+   "metadata": {},
+   "source": [
+    "The high-level `Mean` transform can be used to aggregate a distributed dataset across an existing dimension or dimensions, similar to Xarray's `.mean()` method:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "68d6f586",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import apache_beam as beam\n",
+    "import numpy as np\n",
+    "import xarray_beam as xbeam\n",
+    "import xarray\n",
+    "\n",
+    "ds = xarray.tutorial.load_dataset('air_temperature')"
```

Comment:
I think printing the ds would be helpful to show the reader that the code below takes an average – especially to those who are not familiar with the Xarray tutorial.

Reply:
done